ocaml/idl/datamodel_errors.ml

```diff
@@ -115,6 +115,8 @@ let _ =
   (* Session errors *)
   error Api_errors.session_authentication_failed []
     ~doc:"The credentials given by the user are incorrect, so access has been denied, and you have not been issued a session handle." ();
+  error Api_errors.session_authorization_failed ["username"; "msg"]
+    ~doc:"User was successfully authenticated, but could not be authorized" ();
```

Comment:
I would try to follow the message structure and vocabulary of the previous error, similar to
"The credentials given by the user do not give them enough permissions, so access has been denied."